src/css/_conn-versation.scss

```diff
@@ -36,15 +36,15 @@
   }
 
   .port {
-    background: $blue-light;
+    background: #65bef6;
   }
 
   .responder {
     .ip {
-      background: $green;
+      background: var(--green);
     }
     .port {
-      background: $green-light;
+      background: #5ec4a8;
```

Comment:
Do we not want to capture this (and all of the zeek log type colors for that matter) in more centralized variables as well?

Reply:
Good point. The Zeek log type colors are mostly centralized in this file: https://github.com/brimsec/brim/pull/1405/files#diff-6764dbaa5ca14068a22c350c9eea02ca7e3848ccdcfee72e8b1348a48ac5333d

The reason I didn't make this light green a variable is because I'm planning on re-doing that ConnVersation component at some point soon. It doesn't really fit the new detail pane width anymore.